src/runtime/String.cpp

```diff
@@ -970,7 +970,7 @@ void* UTF16String::operator new(size_t size)
     return GC_MALLOC_EXPLICITLY_TYPED(size, descr);
 }
 
-String* String::getSubstitution(ExecutionState& state, String* matched, String* str, size_t position, StringVector& captures, Value namedCapture, String* replacement)
+String* String::getSubstitution(ExecutionState& state, String* matched, String* str, size_t position, StringVector& captures, Object* namedCapture, String* replacement)
```

Comment:
Why did you change the type of `namedCapture` parameter?
I think that `Value` type is more suitable because standard also handle it as Value type.

Reply:
I  revert my changes on getSubstitution method